metadata.yaml

```diff
@@ -13,21 +13,19 @@ name: content-cache
 display-name: Content Cache
 
 # (Required)
-summary: A very short one-line summary of the charm.
+summary: A charm for deploying and managing a static web content cache with nginx.
 docs: https://discourse.charmhub.io
 issues: https://github.com/canonical/is-charms-template-repo/issues
 maintainers: 
   - https://launchpad.net/~canonical-is-devops
 source: https://github.com/canonical/is-charms-template-repo
 
 description: |
-  A single sentence that says what the charm is, concisely and memorably.
+  A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) deploying and managing a static web content cache with nginx.
 
-  A paragraph of one to three short sentences, that describe what the charm does.
-
-  A third paragraph that explains what need the charm meets.
-
-  Finally, a paragraph that describes whom the charm is useful for.
+  This machine charm manages a nginx instance configured as a content cache. 
```

Comment:
We should probably explain a bit what we mean by "Content cache"

Reply:
Updated.